src/java/io/bazel/rulesscala/jdeps/BUILD

```diff
@@ -0,0 +1,11 @@
+load("@rules_java//java:defs.bzl", "java_library")
+
+java_library(
+    name = "jdeps",
+    srcs = ["JdepsWriter.java"],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@bazel_tools//src/main/protobuf:deps_java_proto",
```

Comment:
Is it safe to depend on this? I do remember there were some issues with `@bazel_tools//src/main/protobuf:worker_java_proto` but I don't remember what was that and maybe this doesn't apply here.

Reply:
I had some quirks with scala toolchain resolution in other parts when I've tried it on dependecy analyzer, but that seems to be fixed in Bazel 5, and current setup seems to work fine with Bazel 4 (I've tested on a large code base)